src/WebJobs.Script/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -70,9 +70,8 @@ public static TValue GetRequestPropertyOrDefault<TValue>(this HttpRequestMessage
 
         public static IDictionary<string, string> GetRawHeaders(this HttpRequestMessage request)
         {
-            Dictionary<string, string> headers = new Dictionary<string, string>();
-
-            var allHeadersRaw = request.Headers.ToString();
+            var headers = new Dictionary<string, string>();
+            var allHeadersRaw = request.Headers.ToString() + request.Content?.Headers?.ToString();
```

Comment:
When concatenating these to ToString() values, is the first guaranteed to have a separator so they don't munge together?